lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2548,7 +2554,7 @@ static void finishPropertyWrapperImplInfo(VarDecl *var,
   }
 
   if (wrapperSetterIsUsable)
-    info = StorageImplInfo::getMutableComputed();
+    info = StorageImplInfo::getMutableOpaque(OpaqueReadOwnership::Owned);
```

Comment:
I think `StorageImplInfo(ReadImplKind::Get, WriteImplKind::Set, ReadWriteImplKind::Modify)` would probably be clearer here.

Reply:
Sure, done.